extension/firefox/chrome/ShumwayCom.jsm

```diff
@@ -273,6 +291,98 @@ var ShumwayCom = {
           return;
         }
         onSystemResourceCallback = callback;
+      },
+      createLocalConnection: function(connectionName, callback) {
+        if (typeof connectionName !== 'string' ||
+            connectionName[0] !== '_' && connectionName.split(':').length !== 2) {
+          return -1; // LocalConnectionConnectResult.InvalidName
+        }
+        if (localConnectionsRegistry[connectionName]) {
+          return -2; // LocalConnectionConnectResult.AlreadyTaken
+        }
+        var connection = {
+          callback,
```

Comment:
Good point. Is there any reason not to use these features, though? They've been available a few versions back already, and all of this code is Firefox-only anyway? We should probably make explicit decisions on which features to use, but I do think that some are just too nice to pass on them.